wagtail/contrib/settings/forms.py

```diff
@@ -17,9 +17,17 @@ def media(self):
     def __init__(self, current_site, model, **kwargs):
         initial_data = {'site': self.get_change_url(current_site, model)}
         super().__init__(initial=initial_data, **kwargs)
-        sites = [(self.get_change_url(site, model), site)
-                 for site in Site.objects.all()]
-        self.fields['site'].choices = sites
+        self.fields["site"].choices = [
+            (
+                self.get_change_url(site, model),
+                (
+                    site.hostname + " [default]"
```

Comment:
Do we need to pass this `default` through a translation?

Reply:
> 1. Confirming that this will supersede #5891 ?
Yes.

> 2. On that other PR you suggested that the default site still always be at the top and then sorted all others after, how come the change?

Because When I looked at it is seemed silly. Mainly because the site list has a column for default. Breaking out alphabetical order, just seems confusing. So I decided to keep it simple.